elyra/util/archive.py

```diff
@@ -114,13 +115,16 @@ def tar_filter(tarinfo):
     matched_set = set()
     temp_dir = create_project_temp_dir()
     archive = os.path.join(temp_dir, archive_name)
+    wildcard_expression_list = ["{WILDCARDS[0]}.py", "{WILDCARDS[0]}.r"]  # Supported script file extensions
```

Comment:
I think you meant for these to be f-strings.
```suggestion
    wildcard_expression_list = [f"{WILDCARDS[0]}.py", f"{WILDCARDS[0]}.r"]  # Supported script file extensions
```

Reply:
Thx, I'll include that! I didn't know I needed it but I believe you 😅 Just curious, is it because I'm using an expression with `{}`?

Reply:
Yes.  Without the `f` prefix that instructs the interpreter to perform the substitution, it will be treat `"{WILDCARDS[0]}.py"` as a string consisting of those characters.